Comanche055/ANGLFIND.agc

```diff
@@ -545,7 +545,7 @@ DELCOMP		SETPD	PUSH		# MPAC CONTAINS THE ANGLE A
 #	 0
 #	C =-COS(THETA)SIN(PSI)COS(PHI)+SIN(THETA)SIN(PHI)
 #	 1
-#	C =COS(THETA)SIN(PSI)SIN(PHI)+SIN(THETA)COS(PHI)
+#	C =COS(THETA)SIN(PSI)SIN(PHI) + SIN(THETA)COS(PHI)
```

Comment:
✨ Change `SIN` to `S N`

![](https://i.imgur.com/B1b0hDs.png)

Reply:
✨ 6th - replace`(` with a space in `COS(THETA)`

![](https://i.imgur.com/wAeWIC6.png)